squad-create-reproducer

```diff
@@ -109,16 +130,31 @@ def run(raw_args=None):
             args.search_build_count,
             args.filename,
             args.allow_unfinished,
+            args.local,
         )
-        reproducer_txt = pathlib.Path(tuxrun_reproducer_file).read_text(
-            encoding="utf-8"
-        )
-        print(reproducer_txt)
     except squadutilslib.ReproducerNotFound as e:
         logger.warning(
             f"No reproducer could be found for {args.group} {args.project} {args.device_name} {args.build_names}"
         )
         logger.warning(f"{e}")
+        return -1
```

Comment:
Should this be logger.warning or logger.error when we return -1 ?

Reply:
Good point - changed.